aggregator/elem_base.go

```diff
@@ -74,6 +74,16 @@ type createAggregationOptions struct {
 	initSourceSet bool
 }
 
+// IDMutationType configs if the id should be mutated after aggregation.
+type IDMutationType int
+
+const (
+	// IDMutationDisabled disables id mutation after aggregation.
+	IDMutationDisabled IDMutationType = iota
```

Comment:
Why not have `IDMutationEnabled` as the default to maintain backward compatibility?